exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -17,27 +17,118 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.ServerSocket;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 import java.util.concurrent.TimeUnit;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
These changes don't seem to be in scope for this PR, could you please revert this file and the others if that's the case?

Reply:
@zoercai these are just the changes in #34. I based off of there so I could get a defined URL to my built jar 😉. Once that's merged I can rebase this and they'll all disappear! Sorry for the confusion, thought I had left this in Draft, my bad.